packages/schematics/src/utils/strings.ts

```diff
@@ -0,0 +1,152 @@
+/**
```

Comment:
This file does similar things to fileUtils. Since you added this one, you should reconcile the two.